plugins/gatsby-source-jenkinsplugins/utils.js

```diff
@@ -201,13 +199,20 @@ const fetchPluginData = async ({createNode, createContentDigest, createNodeId, c
     const documentationListUrl = 'https://updates.jenkins.io/current/plugin-documentation-urls.json';
     const documentation = await requestGET({url: documentationListUrl, reporter});
 
+    const canonicalLabelPairs = await getCoreResourceAsTuples('jenkins/canonical-labels.txt', ' ', reporter);
```

Comment:
oh man at some point I really want to pull these apart into separate graphql objects now that I know how to @link objects, its gotten a little messy over the years :)